cfgov/jinja2/v1/landing-page-1/index.html

```diff
@@ -52,8 +53,19 @@
     </div>
 {% endblock %}
 
+{% block content_sidebar_modifiers -%}
+    o-content-sidebar
```

Comment:
The class is being added in order to target it's descendants in `cfgov/preprocessed/css/organisms/content-sidebar.less`